fhir-bulkimportexport-webapp/src/main/java/com/ibm/fhir/jbatch/bulkdata/export/group/ChunkReader.java

```diff
@@ -108,10 +111,13 @@ private Group findGroupByID(String groupId) throws Exception{
         searchContext.setPageSize(pageSize);
         FHIRTransactionHelper txn = new FHIRTransactionHelper(fhirPersistence.getTransaction());
 
-        txn.enroll();
-        persistenceContext = FHIRPersistenceContextFactory.createPersistenceContext(null, searchContext);
-        patients = fhirPersistence.search(persistenceContext, Patient.class).getResource();
-        txn.unenroll();
+        txn.begin();
+        try {
+            persistenceContext = FHIRPersistenceContextFactory.createPersistenceContext(null, searchContext);
+            patients = fhirPersistence.search(persistenceContext, Patient.class).getResource();
+        } finally {
+            txn.end();
+        }
```

Comment:
not sure if this can work well, need to be verified by the minio enabled build pipeline.